src/puppetlabs/trapperkeeper/services/authorization/authorization_core.clj

```diff
@@ -79,6 +86,31 @@
       (add-acl m)
       (add-query-params m)))
 
+(defn in-valid-methods?
+  "Is the given method contained in the list of valid methods?"
+  [m]
+  (let [method (str/lower-case m)]
+    (boolean (some #(= method %) valid-methods))))
```

Comment:
Let's make the `valid-methods` a set and then we can probably get rid of this helper function